src/GraphQL.Conventions/Types/Resolution/Extensions/ReflectionExtensions.cs

```diff
@@ -64,6 +64,7 @@ public static bool IsEnumerableGraphType(this TypeInfo type)
             return type.IsGenericType(typeof(List<>)) ||
```

Comment:
@K-Pavlov - Hmm... I kinda feel like this entire method could be boiled down to a single expression... Wouldn't checking for the type's interfaces collection for `IEnumerable` be sufficient here? Something along the lines of: `type.GetInterfaces().Any(x => x == typeof(IEnumerable))` or something like that?

Looks OK for now if it fixes your issue for the time being I guess. 👍